jsEx/formForJs.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta content="text/html; charset=UTF-8" http-equiv="Content-Type" />
+    <script src="Form.js"></script>
+</head>
+<body>
+    <form action="register.php" onsubmit="return /*validate()*/ false" method="POST">
+        <label for="username">Потребителско име</label>
+        <input type="text" id="username" name="username">
+        <label id="invalidUsername"></label>
+        <br>
+        <label for="password">Парола</label>
+        <input type="text" id="password" name="password">
```

Comment:
The password should be hidden visually from the user and people around him. Please use type password instead of text.